### ISSUE TITLE
Nginx does not bind to port 3000

### ISSUE THREAD

**Issue by awptechnologies** [linuxserver/docker-remmina#12]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I have ipv6 disabled for all of my devices and i believe this is why port 3000 wont bind. It shows [::]:3000 error.

What file can i reconfigure to make nginx only listen on ipv4 addres..

So far i have edited the .conf in /defailts
i edited sites-available default

i attached a volume to keep my changes persistent but no luck

I feel like there is a hidden config that keeps pushing the configuration to nginx when container starts, regaurdless of what i put in the directories i think are related to nginx.

### Steps To Reproduce

run container

### Environment

```markdown
- OS:
- How docker service was installed:
Debian
```

### CPU architecture

x86-64

### Docker creation

```bash
docker compose up -d
```

### Container logs

```bash
I dont have them rn i will get later if needed. If someone would just tell me where nginx gets its config from i believe i can fix it...
```

**Comment by awptechnologies** [comment]
i figured it out. I tried it again tonight. I mapped the /defaults folder to a volume started container so it would populate volume. Removed the stack. Edited default.conf by removing the 2 listening ports associated with ipv6. Saved the config restarted the container and it seems to have loaded the config to available-sites and is working as expected. Just wanted to post in-case someone else hates ipv6 as much as me.
